src/services/logqlMatchers.ts

```diff
@@ -89,8 +103,82 @@ export function getMatcherFromQuery(query: string): Filter[] {
       type: selectorPosition.contains(matcherPosition) ? LabelType.Indexed : undefined,
     });
   }
+}
+
+function parseLineFilters(query: string, lineFilters: LineFilterType[]) {
```

Comment:
Same question about unit tests, direct or indirect, for these functions.

Reply:
same as the scenes methods above, covered by tests in `links.test.ts` and e2e